modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMMDFRequest.path

```diff
@@ -68,6 +68,20 @@
 <tr>
 	<td>DETAIL_FIELD</td>
 	<td>//td/*[contains(text(),'${key_fieldName}')]//ancestor::tr//*[contains(text(),'${key_nameValue}')]</td>
+</tr>
+<tr>
+	<td>APPLY_BUTTON</td>
+	<td>//div[contains(@class,'drilldown-current')]//button[contains(@class,'btn-sm btn-primary') and contains(text(),'Apply')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>FILTER_TYPE</td>
+	<td>//div[contains (@class,'dropdown-menu') and contains(@class,'show')]//button/span[text()='${keyPartnerOptions}']</td>
```

Comment:
Hi @juliosilveira77, we usually follow a pattern to define variables on paths: We use snake case + camel case, like this: key_variableName. So your path should be key_partnerOptions.